config/eslint/frontend/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  env: {
+    es6: true,
+    browser: "true",
+    "jest/globals": true
+  },
+  extends: [
+    "plugin:styled-components-a11y/recommended",
+    "plugin:prettier/recommended",
+    "react-app",
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "cypress"
+  ],
+  globals: {},
+  plugins: ["react", "react-hooks", "@emotion", "jest"],
+  rules: {
+    "react/jsx-filename-extension": 0,
+    "react/jsx-one-expression-per-line": 0,
+    "react/jsx-props-no-spreading": 0,
+    "react/destructuring-assignment": [2, "always"],
+    "react/no-deprecated": 2,
+    "react/no-unsafe": 2,
+    "react/no-unused-state": 2,
+    "react/jsx-pascal-case": 2,
+    "react/react-in-jsx-scope": 0,
+    "react-hooks/exhaustive-deps": 0,
+    "@emotion/syntax-preference": [2, "object"],
+    "jsx-a11y/no-static-element-interactions": 0,
+    "react/require-default-props": "off",
+    "no-nested-ternary": "off",
+    "prettier/prettier": 1,
```

Comment:
I believe 30-32 came from the Fandom snippet
- `require-default-props` - prefer off, but open to discuss
- `no-nested-ternary` - prefer off, but open to discuss
- `prettier/prettier` - were we thinking this maybe should not be a linting rule, but instead a pre-commit hook? I'm definitely fine with it as a warning here. I'd be open to setting it to error, but sometimes that can feel tedious when testing locally

Reply:
- agreed on `require-default-props`
- `no-nested-ternary` - I do tend to think that nested ternaries make logic more difficult to parse in general, but don't feel too strongly about this if others want to leave this disabled.
- `prettier/prettier` - I think we can leave this as a warning. From what I can tell in [the docs](https://github.com/prettier/eslint-plugin-prettier#recommended-configuration) `plugin:prettier/recommended` should handle separating prettier and eslint for us.

Reply:
I'll +1 to `no-nested-ternary` off, but I could also go either way on it. The way prettier formats them helps slightly with readability.

In my opinion cluttered if/else logic is almost as bad, but admittedly a bit less confusing than nested ternaries.